.gitignore

```diff
@@ -1,3 +1,4 @@
 minio-operator
 !minio-operator/
 .idea/
+dist/
```

Comment:
Why did this show up?

Reply:
goreleaser - release tool creates `dist/` directory - we should ignore that.